### PR TITLE
edfbrowser: fixed icon

### PR DESCRIPTION
**Pull request by whitone**
This PR optimizes the icon by redrawing original PNG icon in Icon-O-Matic.

The previous version of the icon was derivated from the SVG automatically converted from the original PNG.

### CODE REVIEW

File: sci-biology/edfbrowser/additional-files/edfbrowser.rdef.in [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+resource app_version {
+       major  = @MAJOR@,
+       middle  = @MIDDLE@,
+
+       variety = B_APPV_FINAL,
+       internal = 0,
+
+       short_info = "EDFbrowser",
+       long_info  = "An universal viewer for medical timeseries storage files"
```
Comment (humdingerb) [inline sci-biology/edfbrowser/additional-files/edfbrowser.rdef.in:11]:
```suggestion
       long_info  = "A universal viewer for medical timeseries storage files"
```

File: sci-biology/edfbrowser/edfbrowser-1.73.recipe [hunk 2]
```diff
@@ -0,0 +1,70 @@
+SUMMARY="An universal viewer for medical timeseries storage files"
```
Comment (humdingerb) [inline sci-biology/edfbrowser/edfbrowser-1.73.recipe:1]:
```suggestion
SUMMARY="A universal viewer for medical timeseries storage files"
```

File: sci-biology/edfbrowser/edfbrowser-1.73.recipe [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+{
+	local MAJOR="`echo "$portVersion" | cut -d. -f1`"
+	local MIDDLE="`echo "$portVersion" | cut -d. -f2`"
+	sed \
+		-e "s|@MAJOR@|$MAJOR|" \
+		-e "s|@MIDDLE@|$MIDDLE|" \
+		$portDir/additional-files/edfbrowser.rdef.in > $sourceDir/edfbrowser.rdef
+
+	mkdir -p $appsDir
+
```
Comment (humdingerb) [inline sci-biology/edfbrowser/edfbrowser-1.73.recipe:64]: Completely cosmetical, I'd remove this empty line.

File: sci-biology/edfbrowser/additional-files/edfbrowser.rdef.in (hunk 1 above)
Comment (whitone) [reply sci-biology/edfbrowser/additional-files/edfbrowser.rdef.in]: Done!

File: sci-biology/edfbrowser/edfbrowser-1.73.recipe (hunk 2 above)
Comment (whitone) [reply sci-biology/edfbrowser/edfbrowser-1.73.recipe]: Done!

File: sci-biology/edfbrowser/edfbrowser-1.73.recipe (hunk 3 above)
Comment (whitone) [reply sci-biology/edfbrowser/edfbrowser-1.73.recipe]: Done!

**Comment by whitone** [comment]
Added icon file in Icon-O-Matic format

**Comment by humdingerb** [comment]
Thanks! LGTM, aside from the conflicts. I assume you made these latest changes on top of the already merged recipe which PR was merged recently...

**Comment by whitone** [comment]
> Thanks! LGTM, aside from the conflicts. I assume you made these latest changes on top of the already merged recipe which PR was merged recently...

Yeah, you're right: now it's fixed!

**Comment by humdingerb** [comment]
OK, one more thing: you'll have to increse the REVISION in the recipe to have the package rebuilt, so that the improved icon/text hits HaikuDepot.

**Comment by whitone** [comment]
> OK, one more thing: you'll have to increse the REVISION in the recipe to have the package rebuilt, so that the improved icon/text hits HaikuDepot.

I supposed I needed to bump the revision, and now it's done!

**Comment by whitone** [comment]
Why Travis CI says that there's a pending check, but when I click on details everything is passed?!?

**Comment by humdingerb** [comment]
No idea what's the issue with Travis... merging. Thanks!

**Pull request merged**